data/json/effects.json

```diff
@@ -458,6 +458,31 @@
             "vomit_chance": [500]
         }
     },
+	{
+        "type": "effect_type",
+        "id": "ooze_in_eyes",
+        "name": ["Gunk in eyes"],
+        "desc": ["There is some sort of substance in your eyes. You can barely see anything now!"],
+        "apply_message": "Your eyes sting and you can keep them barely open, as some sort of liquid splashes into them!",
```

Comment:
The second part of the message shouldn't be in apply, but in the monster attack code.